From: Paul Blakey <paulb@mellanox.com>
Date: Wed, 25 Sep 2019 18:02:35 +0300
Subject: net/sched: Set default of CONFIG_NET_TC_SKB_EXT to N
Patch-mainline: v5.4-rc1
Git-commit: dfe5999dc03e321d08190772c98843284d5cf419
References: bsc#1176447

This a new feature, it is preferred that it defaults to N.
We will probe the feature support from userspace before actually using it.

Fixes: 95a7233c452a ('net: openvswitch: Set OvS recirc_id from tc chain index')
Signed-off-by: Paul Blakey <paulb@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 net/sched/Kconfig |    1 -
 1 file changed, 1 deletion(-)

--- a/net/sched/Kconfig
+++ b/net/sched/Kconfig
@@ -966,7 +966,6 @@ config NET_IFE_SKBTCINDEX
 config NET_TC_SKB_EXT
 	bool "TC recirculation support"
 	depends on NET_CLS_ACT
-	default y if NET_CLS_ACT
 	select SKB_EXTENSIONS
 
 	help
